codalab/model/mysql_model.py

```diff
@@ -44,7 +44,7 @@ class MySQLModel(BundleModel):
     def __init__(self, engine_url, default_user_info):
         if not engine_url.startswith('mysql://'):
             raise UsageError('Engine URL should start with mysql://')
-        engine = create_engine(engine_url, strategy='threadlocal', pool_size=20, max_overflow=100, pool_recycle=3600)
+        engine = create_engine(engine_url, strategy='threadlocal', pool_size=20, max_overflow=100, pool_recycle=3600, encoding='utf-8')
```

Comment:
Does this mean that we don't need to call `encode_str` and `decode_str` and can pass unicode strings directly into DB operations?